internal/provider/resources/passwordconfig.go

```diff
@@ -170,12 +170,6 @@ func (r passwordConfigResource) ValidateConfig(ctx context.Context, req resource
 		return
 	}
 
-	// If the projectID isn't yet known, skip validation for now.
```

Comment:
FWIW, I only commented once but there's at least one other instance of this pattern that could be eliminated.